docs/api/api.md

```diff
@@ -0,0 +1,397 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# API doc of Manager API.
+It's used to manage etcd and provide APIs to the frontend interface.
+
+## Version: 2.2
+
+**License:** [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+
+### /apisix/admin/check_ssl_cert
+
+#### POST
+##### Summary:
+
+verify SSL cert and key
+
+##### Description:
+
+verify SSL cert and key
+
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| cert | body | cert of SSL | Yes | string |
+| key | body | key of SSL | Yes | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | SSL verify passed | [ApiError](#ApiError) |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### /apisix/admin/check_ssl_exists
+
+#### POST
+##### Summary:
+
+check SSL exists or not
+
+##### Description:
+
+check SSL exists or not by sni
+
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| cert | body | cert of SSL | Yes | string |
+| key | body | key of SSL | Yes | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | SSL exists | [ApiError](#ApiError) |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### /apisix/admin/consumers
+
+#### GET
+##### Summary:
+
+Returns consumer list
+
+##### Description:
+
+Return the consumer list according to the specified page number and page size, and can search by username
```

Comment:
![image](https://user-images.githubusercontent.com/6814606/101596805-ba196600-3a30-11eb-957e-9d191c9c2a64.png)

Reply:
`and can search by username`
to
`and support to search "consumer" by username`

Reply:
fixed.